implementations/rust/ockam/ockam_vault/Cargo.toml

```diff
@@ -73,6 +72,7 @@ storage = ["ockam_node/storage", "sqlx"]
 [dependencies]
 aes-gcm = { version = "0.10", default-features = false, features = ["aes"] }
```

Comment:
Should we exclude that dependency if aws-lc is used?

Reply:
I couldn't find a way to use “negative” dependency.
Either you make the library user one of the two features (`aws-lc` or `aes-gcm`) or you always build the default one.